src/pages/resultsView/enrichments/MiniScatterChart.tsx

```diff
@@ -73,6 +79,22 @@ export default class MiniScatterChart extends React.Component<
         }
     }
 
+    private get genericAssayEntityTitle() {
+        if (this.isGenericAssay && this.props.genericAssayType) {
+            return (
+                GENERIC_ASSAY_CONFIG.genericAssayConfigByType[
+                    this.props.genericAssayType
+                ]?.globalConfig?.entityTitle ||
+                deriveDisplayTextFromGenericAssayType(
+                    this.props.genericAssayType
+                )
+            );
+        } else {
+            // This code won't be reached
```

Comment:
?

Reply:
I tried to satisfy the ts checking, I will replace it with
```
return '';
```